hasher-matcher-actioner/webapp/src/pages/SubmitContent.jsx

```diff
@@ -75,26 +76,39 @@ export default function SubmitContent() {
   const handleSubmit = event => {
     event.preventDefault();
     setSubmitting(true);
+    setSubmissionError(false);
     if (inputs.submissionType === 'PUT_URL_UPLOAD') {
       submitContentViaPutURLUpload(
         inputs.contentId,
         inputs.contentType,
         packageAdditionalFields(),
         inputs.content.raw,
-      ).then(() => {
-        setSubmitting(false);
-        setSubmittedId(inputs.contentId);
-      });
+        inputs.resubmit,
```

Comment:
nit: Does not exist in FORM_DEFAULTS. That means inadvertent type coercion of undefined into false.

Reply:
good catch! thanks!